x-pack/plugins/canvas/public/components/sidebar_header/sidebar_header.tsx

```diff
@@ -0,0 +1,329 @@
+
+    return {
+      name: 'Order',
+      className: topBorderClassName,
+      panel: {
+        id: 1,
+        title: 'Order',
+        items: [
+          {
+            name: 'Bring to front', // TODO: check against current element position and disable if already top layer
+            icon: 'sortUp',
+            onClick: () => bringToFront(),
+          },
+          {
+            name: 'Bring forward', // TODO: same as above
+            icon: 'arrowUp',
+            onClick: () => bringForward(),
+          },
+          {
+            name: 'Send backward', // TODO: check against current element position and disable if already bottom layer
+            icon: 'arrowDown',
+            onClick: () => sendBackward(),
+          },
+          {
+            name: 'Send to back', // TODO: same as above
+            icon: 'sortDown',
+            onClick: () => sendToBack(),
+          },
+        ],
+      },
+    };
+  };
+
+  // TODO: restore when group and ungroup can be triggered outside of workpad_page
+  // private _getGroupMenuItem = () => {
+  //   const { groupIsSelected, ungroupNodes, groupNodes } = this.props;
+  //   return groupIsSelected
+  //     ? {
+  //         name: 'Ungroup',
+  //         className: topBorderClassName,
+  //         onClick: () => {
+  //           ungroupNodes();
+  //         },
+  //       }
+  //     : {
+  //         name: 'Group',
+  //         className: topBorderClassName,
+  //         onClick: () => {
+  //           groupNodes();
+  //         },
+  //       };
+  // };
+
+  private _renderPanelTree = (closePopover: () => void) => {
+    const {
+      showLayerControls,
+      cutNodes,
+      copyNodes,
+      pasteNodes,
+      deleteNodes,
+      cloneNodes,
+    } = this.props;
+
+    const items = [
+      {
+        name: 'Cut',
+        icon: 'cut',
+        onClick: () => {
+          cutNodes();
+          closePopover();
+        },
+      },
+      {
+        name: 'Copy',
+        icon: 'copy',
+        onClick: () => copyNodes(),
+      },
+      {
+        name: 'Paste', // TODO: can this be disabled if clipboard is empty?
+        icon: 'copyClipboard',
+        onClick: () => {
+          pasteNodes();
+          closePopover();
+        },
+      },
+      {
+        name: 'Delete',
+        icon: 'trash',
+        onClick: () => {
+          deleteNodes();
+          closePopover();
+        },
+      },
+      {
+        name: 'Clone',
+        onClick: () => {
+          cloneNodes();
+          closePopover();
+        },
+      },
+      // TODO: restore when group and ungroup can be triggered outside of workpad_page
+      // this._getGroupMenuItem(),
+    ];
+
+    if (showLayerControls) {
+      // @ts-ignore - this is the right shape of an EUI panel tree
+      items.push(this._getLayerMenuItems());
+    }
+
+    items.push({
+      name: 'Save as new element',
+      icon: 'indexOpen',
+      // @ts-ignore - this is a valid prop
```

Comment:
...or you can fix this by creating a type for `items` on 207.  TS is complaining that you're trying to push an item whose shape does not match the inferred type from the collection above, because none have `className`.  If you define the `items` array on 207 to be the following, this warning will go away.

```
interface Item {
  name: string;
  icon: string;
  onClick: () => void;
  className?: string;
}

const items: Item[] = ...
```

Reply:
I was able to borrow type defs from `@elastic/eui` to fix this. 👍